src/features/call/components/PreviousCallsSection.tsx

```diff
@@ -0,0 +1,196 @@
+import { Box } from '@mui/material';
+import { FC } from 'react';
+
+import useCallMutations from '../hooks/useCallMutations';
+import useOutgoingCalls from '../hooks/useOutgoingCalls';
+import ZUIPersonAvatar from 'zui/components/ZUIPersonAvatar';
+import ZUIText from 'zui/components/ZUIText';
+import ZUIButton from 'zui/components/ZUIButton';
+import ZUIDivider from 'zui/components/ZUIDivider';
+import { labels, colors } from './PreviousCallsInfo';
+import { ZetkinCall } from '../types';
+import ZUIRelativeTime from 'zui/ZUIRelativeTime';
+
+type PreviousCallsSectionProps = {
+  assingmentId: number;
+  onClose?: () => void;
+  onSwitchCall?: () => void;
+  orgId: number;
+  searchTerm?: string;
+};
+
+const PreviousCallsSection: FC<PreviousCallsSectionProps> = ({
+  assingmentId,
+  onClose,
+  onSwitchCall,
+  orgId,
+  searchTerm,
+}) => {
+  assingmentId;
+  const { deleteCall, switchCurrentCall } = useCallMutations(orgId);
+  const outgoingCalls = useOutgoingCalls();
+  const search = searchTerm?.toLowerCase().trim();
+
+  const matchesSearch = (call: ZetkinCall) => {
+    if (!search) {
+      return true;
+    }
+    const name =
+      `${call.target.first_name} ${call.target.last_name}`.toLowerCase();
+    const phone = call.target.phone?.toLowerCase() || '';
+
+    const searchParts = search.split(' ').filter((part) => part !== '');
+
+    return searchParts.every(
+      (part) => name.includes(part) || phone.includes(part)
+    );
+  };
```

Comment:
in some places we use the `fuse` library to do searching, this could be used here I think to - but nothing that stops it from being merged now, I searched for name and number etc and it works well!